src/typegraph/core/src/logger.rs

```diff
@@ -4,52 +4,28 @@
 #[allow(unused)]
 macro_rules! debug {
     ( $($arg:tt)* ) => {
-        {
-            use std::fmt::Write as _;
-
-            let mut msg = "debug: ".to_string();
-            write!(&mut msg, $($arg)*).unwrap();
-            $crate::wit::metatype::typegraph::host::print(&msg);
-        }
+        println!("debug: {}", format!($($arg)*));
```

Comment:
I think we should remove all of these, and use the console actor where it is used. It will be relevant when we re-enable parallel tasks...